restock.lic

```diff
@@ -0,0 +1,166 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#restock
+=end
+
+custom_require.call(%w[common common-items common-money])
+
+$NUM_MAP = {
```

Comment:
It looks like [workorders.lic](https://github.com/rpherbig/dr-scripts/blob/master/workorders.lic#L496) has something similar. Is it time to find a common place for this?

Reply:
done